sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -23,21 +23,27 @@
 import abc
 import contextlib
 import logging
-import Queue as queue
+import queue
 import sys
 import threading
 import traceback
+from builtins import object
+from builtins import range
 from concurrent import futures
 
 import grpc
-import six
+from future import standard_library
+from future.utils import raise_
+from future.utils import with_metaclass
 
 from apache_beam.portability.api import beam_fn_api_pb2
 from apache_beam.portability.api import beam_fn_api_pb2_grpc
 from apache_beam.runners.worker import bundle_processor
 from apache_beam.runners.worker import data_plane
 from apache_beam.runners.worker.worker_id_interceptor import WorkerIdInterceptor
 
+standard_library.install_aliases()
```

Comment:
Do we want to call this before we do the import of "queue" above?  It looks like this only works because of an "accident" in that this module is imported after someone else already called `install_aliases()`.

Reply:
Order should be OK, see: https://github.com/apache/beam/pull/5373#discussion_r200076690